documentation/devonfw-ide-introduction.asciidoc

```diff
@@ -1,12 +1,73 @@
-= Introduction
+= devonfw-ide
 
-http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries
-as well as industry proven practices and code conventions.
+== What is devonfw-ide?
+
+`devondw-ide` is a standardized, open-source toolbox or construction kit allowing you to pick and choose those tools which fit the requirements and needs of your project.
+
+It is not another Eclipse. It gives you Eclipse, along with Java, maven, NodeJs, VSCode and everything else you may need to quick start your development. All you need to do is download and extract a zip file and execute its setup, reducing your initial setup time for starting a project from days to minutes.
+
+That is not all. `devonfw-ide` gives you its own command shell where you can execute `devon` commands. Here is a small video which showcases what you can do with `devonfw-ide`:
+
+[youtube video link]
+
+Or you can also try out `devonfw-ide` in our interactive Katacoda scenario without having to download it at all:
+
+[katacoda tutorial link]
+
+http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries as well as industry proven practices and code conventions.
```

Comment:
Why do we need this? This should only be about devon-ide

Reply:
since it is about devonfw, removed.